packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -140,6 +140,11 @@ export const configSchema = {
     _default: '',
     _description: 'Custom label for patient chart button',
   },
+  showDashboardLinkIcon: {
```

Comment:
Honestly, I don't think this needs to be a configuration option. I'd just check if the link configuration includes an icon.

Reply:
Please remove this config element